lib/components/MenuRenderer/MenuRenderer.tsx

```diff
@@ -205,7 +205,7 @@ export const MenuRenderer = ({
     // first menu item is not highlighted automatically, but considering
     // space keyboard interactions are optional this is acceptable.
     //   See Firefox bug details: https://bugzilla.mozilla.org/show_bug.cgi?id=1220143
-    //   See WAI-ARIA keyboard iteractions: https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-12
+    //   See WAI-ARIA keyboard iteractions: https://www.w3.org/TR/wai-aria-practices-1.2/#keyboard-interaction-12
```

Comment:
Any context to this?

Reply:
A while back i made sure everything aligned to the same version. Noticed all the menu links were to the old spec. So reviewed that there was no change required and bumped our docs links/refs